docs/src/content/docs/utilities/Signals/computed-from.md

```diff
@@ -71,7 +73,8 @@ let c = computedFrom(
 			([a, b]) =>
 				// of(a + b) is supposed to be an asynchronous operation (e.g. http request)
 				of(a + b).pipe(delay(1000)) // delay the emission of the combined value by 1 second for demonstration purposes
-		)
+		),
+		{ initialValue: 42 } // change the initial value of the resulting signal
```

Comment:
I'd like to have this here as is. And add another example where we use initial value to showcase how it changes the output.

Reply:
If I remove the { initialValue: 42 } this sample will THROW becouse it's exactly the case where the input + delay operator generate "late async" emit , so with the new implementation, you don't get the "old spurious" initial emit of array [1,2] (that is what I really don't like because it's totally un-expected (and maybe error prone) with an inferred output as Signal<number>.

This is why I introduced { initialValue: 42 } in the sample, and adjusted the output expected in the console.

Let's me know if you agree, or if you prefer to have a different sample showing the use-case using only inputs + pipe Operator, and keeping this one to show the initialValue to prevent throwing error... 👀

Reply:
Let's have both cases. The one that throws and the new one that doesn't.

Reply:
Ok @eneajaho I added the first sample without params so it'll throw error, and then show the two possible solutions with initialValue or with startWith. 
Let's me know what you think?